src/esthri/Cargo.toml

```diff
@@ -38,26 +37,46 @@ name = "esthri"
 path = "src/lib.rs"
 
 [features]
-default = []
+default = [ "rustls", "cli", "http_server", "blocking", ]
 aggressive_lint = []
+blocking = []
 http_server = [ "async-tar", "async-compression", "async-stream", "bytes", "futures-util", "mime_guess", "sluice", "sanitize-filename", "serde", "serde_derive", "tokio-util", "warp", "ctrlc",]
-nativetls = [ "rusoto_nativetls",]
-rustls = [ "rusoto_rustls",]
-cli = [ "ctrlc", "structopt",]
+nativetls = [ "rusoto_credential", "rusoto_core/native-tls", "rusoto_signature", "rusoto_s3/native-tls", "hyper-tls",]
+rustls = [ "rusoto_credential", "rusoto_core/rustls", "rusoto_signature", "rusoto_s3/rustls", "hyper-rustls",]
```

Comment:
we don't need our own wrapper modules around rusoto to enable features since we can turn on features with the `<dep>/<feature_name>` syntax